snark/src/lib.rs

```diff
@@ -23,7 +23,7 @@ pub trait SNARK<F: PrimeField> {
 
     /// The information required by the verifier to check a proof for a specific
     /// circuit *C*.
-    type VerifyingKey: Clone + ToBytes;
+    type VerifyingKey: Clone;
```

Comment:
Should we replace this with a `CanonicalSerialize + CanonicalDeserialize` bound?

Reply:
(Same for `ProvingKey`)